technical-reports/resolver/introduction.md

```diff
@@ -1,10 +1,3 @@
 # Introduction
 
-Design tokens are key-value pairs that represent design decisions, allowing for consistent use across different platforms and tools. In complex design systems, there is a need to manage multiple dimensions such as themes, modes (e.g., light and dark), brands, and variants. This introduces challenges in organizing tokens, resolving references, and applying overrides.
-
-The Resolver Specification addresses these challenges by introducing a structured mechanism to:
```

Comment:
“The Resolver Specification” is incorrect, it should be “The Resolver module” (or something similar).

Further, we don’t need to summarize how these “challenges” are addressed because we never defined or described how these are problems to begin with, so the solutions are equally nebulous.